doc/how_to/plotly.md

```diff
@@ -0,0 +1,183 @@
+# Plotly
+
+Panel Material UI provides integrated theming support for Plotly. This includes built-in `"pmui_light"` and `"pmui_dark"` [Plotly templates](https://plotly.com/python/templates/).
+
+## Available Templates
+
+When you import `panel_material_ui`, Plotly templates are automatically registered for immediate use:
+
+- `"pmui_light"` - Light theme (set as default)
+- `"pmui_dark"` - Dark theme
+
+Both templates follow Material Design principles with proper typography, spacing, and color schemes.
+
+## Basic Usage
+
+The `"pmui_light"` template is applied by default.
+
+```{pyodide}
+import panel as pn
+import panel_material_ui as pmui
+import plotly.express as px
+
+pn.extension("plotly")
+
+df = px.data.iris()
+
+plot = px.scatter(
+    df, x="sepal_length", y="sepal_width", color="species",
+    height=400
+)
+
+pmui.Container(
+    plot, width_option="md"
+).preview()
+```
+
+You may also apply the `"pmui_dark"` template.
+
+```{pyodide}
+import panel as pn
+import panel_material_ui as pmui
+import plotly.express as px
+
+pn.extension("plotly")
+
+df = px.data.iris()
+
+plot = px.scatter(
+    df, x="sepal_length", y="sepal_width", color="species",
+    height=400, template="pmui_dark"
+)
+
+pmui.Container(
+    plot, width_option="md"
+).preview()
+```
+
+## Reactive Theme Switching
+
+Create plots that automatically respond to theme changes by using the `update_template` function with Panel's reactive system:
```

Comment:
Would be really nice if this was just automatic and we did not have to explain this.

See https://github.com/holoviz/panel/issues/8005

Reply:
Would That hoppen Short term? Or should we document work around for now?